protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -298,3 +298,17 @@ enum CoreOperatorType {
   NESTED_LOOP_JOIN = 35;
   AVRO_SUB_SCAN = 36;
 }
+
+message Func {
```

Comment:
Protobufs are hard to guess at without documentation; there is no code to provide a hint about the meaning of fields. Can you provide a description of how these are used?